jdaviz/configs/cubeviz/plugins/tests/test_cubeviz_aperphot.py

```diff
@@ -7,16 +7,18 @@
 from numpy.testing import assert_allclose
 from regions import RectanglePixelRegion, PixCoord
 
-from jdaviz.core.custom_units import PIX2
+from jdaviz.core.custom_units_and_equivs import PIX2, SPEC_PHOTON_FLUX_DENSITY_UNITS
+from jdaviz.core.unit_conversion_utils import (flux_conversion_general,
+                                               handle_squared_flux_unit_conversions)
 
 
 def test_cubeviz_aperphot_cube_orig_flux(cubeviz_helper, image_cube_hdu_obj_microns):
     cubeviz_helper.load_data(image_cube_hdu_obj_microns, data_label="test")
-    flux_unit = u.Unit("1E-17 erg*s^-1*cm^-2*Angstrom^-1*pix^-2")  # actually a sb
+    flux_unit = u.Unit("1E-17 erg*s^-1*cm^-2*Angstrom^-1*pix^-2")
     solid_angle_unit = PIX2
 
     aper = RectanglePixelRegion(center=PixCoord(x=1, y=2), width=3, height=5)
-    cubeviz_helper.plugins['Subsets'].import_region(aper)
+    cubeviz_helper.plugins['Subsets']._obj.import_region(aper)
```

Comment:
Why add the `._obj` here? This PR should not change behavior of existing plugin API, right?